src/backend/marsha/core/lti/__init__.py

```diff
@@ -53,15 +56,17 @@ def verify(self):
             consumer site with the consumer site domain passed in the LTI request.
 
         """
+        self._is_verified = True
```

Comment:
I think it should be set only at the end of the method. But maybe we don't need `self._is_verifier`. What about just starting this method by:
```python
if self._consumer_site:
    return self._consumer_site
```